File: JavaToKotlin/apps(kotlin)/src/main/kotlin/io/dustin/apps/board/domain/qna/question/repository/custom/CustomQuestionRepository.java
package io.dustin.apps.board.domain.qna.question.repository.custom;

import io.dustin.apps.board.domain.qna.question.model.dto.QuestionDto;

import java.util.List;

public interface CustomQuestionRepository {

    QuestionDto getQuestion(long userId, long questionId);

    List<QuestionDto> getQuestionList(long userId, Long nextId, int size);
}

File: apps/src/main/kotlin/io/dustin/apps/board/domain/qna/question/repository/custom/CustomQuestionRepository.kt
package io.dustin.apps.board.domain.qna.question.repository.custom

import io.dustin.apps.board.domain.qna.question.model.dto.QuestionDto

interface CustomQuestionRepository {
    fun getQuestion(userId: Long, questionId: Long): QuestionDto?
    fun getQuestionList(userId: Long, recordsCount: Long, nextId: Long?): List<QuestionDto>
}
